packages/camera/camera/test/camera_preview_test.dart

```diff
@@ -133,8 +133,11 @@ void main() {
         isInitialized: true,
         isRecordingVideo: true,
         deviceOrientation: DeviceOrientation.portraitUp,
-        lockedCaptureOrientation: DeviceOrientation.landscapeRight,
-        recordingOrientation: DeviceOrientation.landscapeLeft,
+        lockedCaptureOrientation:
+            const Optional<DeviceOrientation>.fromNullable(
+                DeviceOrientation.landscapeRight),
+        recordingOrientation: const Optional<DeviceOrientation>.fromNullable(
+            DeviceOrientation.landscapeLeft),
```

Comment:
i'm a bit confused about this part - is `Optional` the old "nullable" before null safety in dart?